.github/workflows/pytest-workflow.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       matrix:
         nxf_version: ['20.11.0-edge']
         tags: ['${{ fromJson(needs.changes.outputs.modules) }}']
-        profile: ['docker', 'singularity'] ## 'conda'
+        profile: ['docker', 'singularity', 'conda']
```

Comment:
Is this going to break everything else once it's in master though?

Reply:
Shouldn't do. Some tests may fail for incoming PRs but we can just approve and merge them anyway. I am hoping this will twist our arm and help us find a permanent fix.